### PR TITLE
[#1328] Add checkboxes for cases, plans, messages notifications to Profile page

### PR DESCRIPTION
**Pull request by vaszig**
I do not have a design for this functionality so I added this to the profile page (in the same way we do the selected categories)

image

image

### CODE REVIEW

File: src/open_inwoner/accounts/urls.py [hunk 1]
```diff
@@ -101,6 +102,7 @@
     path("contacts/", ContactListView.as_view(), name="contact_list"),
     path("onderwerpen/", MyCategoriesView.as_view(), name="my_categories"),
     path("mydata/", MyDataView.as_view(), name="my_data"),
+    path("notifications/", MyNotificationsView.as_view(), name="my_notifications"),
```
Comment (Bartvaderkin) [inline src/open_inwoner/accounts/urls.py:105]: I just like to note this 'My' naming pattern is awful but I'll accept it as at least it keeps the pattern.

File: src/open_inwoner/accounts/tests/test_notify_about_messages.py [hunk 2]
```diff
@@ -31,6 +31,21 @@ def test_notify_about_received_message(self):
...
+        self.assertFalse(message.sent)
+
+        call_command("notify_about_messages")
+
+        self.assertEqual(len(mail.outbox), 0)
+
+        message.refresh_from_db()
+
+        self.assertFalse(message.sent)
+
```
Comment (Bartvaderkin) [inline src/open_inwoner/accounts/tests/test_notify_about_messages.py:48]: Not an issue but notably different whitespace compared to the others :wink:

**Review by Bartvaderkin — APPROVED** [review]
Neat!

File: src/open_inwoner/templates/pages/profile/notifications.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% extends 'master.html' %}
+{% load i18n l10n form_tags %}
+
+{% block content %}
+<h1 class="h1" id="title">
+    {% trans "Mijn notificaties" %}
+</h1>
+<p class="p">
+    {% trans "Kies voor welke categorie je notificaties wil ontvangen" %}
```
[the remark this answers is not in the document]
Comment (alextreme) [reply src/open_inwoner/templates/pages/profile/notifications.html:9]: Lets go for 'meldingen' in the Dutch labels

**Review by alextreme — CHANGES_REQUESTED** [review]
Ensure dutch labels use 'Communicatievoorkeuren' instead of 'mijn notificaties'.

We discussed last week to use 'meldingen' however this term is overloaded.

**Review by jiromaykin — APPROVED** [review]

**Review by alextreme — APPROVED** [review]

**Pull request merged**